spec/models/piece_spec.rb

```diff
@@ -93,5 +93,69 @@
       expect(pawn1.obstructed_horizontal?([4, 1])).to eq false
     end
 
-  end
+  end # end describe obstructed_horizontal
+
+  describe "#obstructed_vertical" do
+
+    before(:example) do
+      @game = create(:game)
+    end
+
+    it "should return true if piece is obstructed above" do
+      pawn1 = @game.pieces.create({
+        color: "white",
+        x_pos: 1,
+        y_pos: 1,
+        captured: false
+      })
+      pawn2 = @game.pieces.create({
```

Comment:
I'm not sure from the docs what you mean. Can we go over this in our call
tomorrow?
On May 1, 2016 4:48 PM, "Kaoru Kohashigawa" <notifications@github.com>
wrote:

> In spec/models/piece_spec.rb
> <https://github.com/kaorus-army/chess_with_friends/pull/4#discussion_r61692846>
> :
>
> > +  end # end describe obstructed_horizontal
> > +
> > +  describe "#obstructed_vertical" do
> > +
> > +    before(:example) do
> > +      @game = create(:game)
> > +    end
> > +
> > +    it "should return true if piece is obstructed above" do
> > +      pawn1 = @game.pieces.create({
> > +        color: "white",
> > +        x_pos: 1,
> > +        y_pos: 1,
> > +        captured: false
> > +      })
> > +      pawn2 = @game.pieces.create({
>
> see how let can help you DRY some of these specs:
> https://relishapp.com/rspec/rspec-core/v/3-3/docs/helper-methods/let-and-let
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/kaorus-army/chess_with_friends/pull/4/files/5757346b0aa531627e97a076d621546b234b2c77#r61692846>
>